### ISSUE TITLE
Building the documentation

### ISSUE THREAD

**Issue by RubendeBruin** [wavespectra/wavespectra#39]
How can the documentation be build locally? I would like to see if my contributions render correctly.

**Comment by rafa-guedes** [comment]
Use [sphinx](https://pypi.org/project/Sphinx/), e.g.

From the docs folder run:
```
sphinx-build -b html ./ _build
```

Then check the `_build/index.html` generated file in a browser

Make sure you have got all the docs [requirements](https://github.com/wavespectra/wavespectra/blob/master/requirements/requirements_doc.txt) installed in your environment.

**Comment by RubendeBruin** [comment]
On windows I had to change

```python
import wavespectra

# sys.path.insert(0, os.path.abspath('.'))
sys.path.insert(0, os.path.abspath("../wavespectra"))
```

to

```python
# sys.path.insert(0, os.path.abspath('.'))
sys.path.insert(0, os.path.abspath("./.."))

import wavespectra
```

to avoid a "can not find package wavespectra" error.

**Comment by rafa-guedes** [comment]
@RubendeBruin the docs build fine with this change both on my linux machine and on RTD, I have pushed it in master. Does that solve your problem on Windows?

**Comment by rafa-guedes** [comment]
Implemented in e50589341bff15d40cfdaed233f6e695fa41cad2
